docs/installation.md

```
@@ -0,0 +1,31 @@
+# Installation Guide
+
+1. Clone the repository:
+   ```
+   git clone https://github.com/testpilot-max/finops.git
+   ```
+
+2. Navigate to the project directory:
+   ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)